conf/services/calculate-stamp-duty-land-tax.yml

```diff
@@ -7,10 +7,43 @@ serviceDescription: |
+    date: 2022-06-21
+  - description: "The ‘Skip to main content’ link does not function as expected and the users focus stays on the link instead of being directed to the main content region of the page. This doesn’t meet WCAG 2.1 success criteria 2.4.1 (bypass blocks) and 2.4.3 (focus order)."
+    date: 2022-06-21
+  - description: "The page titles throughout the service are not unique and are duplicated. This doesn’t meet WCAG 2.1 success criteria 2.4.2 (page titled)."
+    date: 2022-06-21
+  - description: "The heading structure on the page is illogical and contains multiple h1 headings. This issue may affect screen reader users who use headings to navigate and identify different sections of information on the page. This doesn’t meet WCAG 2.1 success criteria 1.3.1 (info and relationships)."
+    date: 2022-06-21
+  - description: "When users activate the ‘Skip to main content’ link on the page they are directed back to the previous page. This doesn’t meet WCAG 2.1 success criteria 2.4.4 (link purpose), 2.4.1 (bypass blocks) and 2.4.9 (link purpose link only)."
+    date: 2022-06-21
+  - description: "Visually, users are able to identify these as headings which introduce new sections of information to the page but they are not marked-up using headings tags for screen reader users who may find difficulty in identifying the sections of information in the same way a sighted user could navigating the page. This doesn’t meet WCAG 2.1 success criteria 1.3.1 (info and relationships)."
+    date: 2022-06-21
+  - description: "On the ‘Check your answers’ page, a visually hidden paragraph has been placed inside each <dt> element in the details list which reads out ‘Provide an answer to continue’ followed by ‘Freehold or leasehold’. As the page is a ‘Check your answers’ page, screen reader users may find the added information confusing as they have already provided the answers in previous pages in the service. This doesn’t meet WCAG 2.1 success criteria 1.3.1 (info and relationships)."
+    date: 2022-06-21
+  - description: "The page has two ‘main’ regions defined by the role=main. This issue may affect screen reader users experience of navigating and identifying different areas of the page. This doesn’t meet WCAG 2.1 success criteria 1.3.1 (info and relationships)."
+    date: 2022-06-21
+  - description: "The ‘How does this work’ details components on the page have duplicate labels. This issue may affect screen reader users who may find difficulty in distinguishing between the two components in their journey. This doesn’t meet WCAG 2.1 success criteria 2.4.6 (headings and labels)."
+    date: 2022-06-21
+serviceLastTestedDate: 2021-06-21
 statementVisibility: public
 statementCreatedDate: 2020-09-18
-statementLastUpdatedDate: 2020-09-18
+statementLastUpdatedDate: 2022-05-11
 businessArea: Customer Services Group (CSG)
 ddc: DDC Telford
 liveOrClassic:
```

Comment:
Does this need setting if included ? Readme suggests : Classic Services, Live Services - Edinburgh, Live Services - Newcastle, Live Services - Telford, Live Services - Worthing

Reply:
That, as far as I can tell has only recently been added and not by me, I have yet to see any guidance on this, official or otherwise in slack channels. During my few weeks doing mainly nothing but accessibility statements, everything below statementLastUpdatedDate has appeared. Seems that @oscarduignan bulk added this here - https://github.com/hmrc/accessibility-statement-frontend/commit/a02aef52bb4b616433b8f752c29645399b848dd2

Reply:
Queried this with plat ui - confirmed that they will require updating at some point and some of which will become mandatory for completion.

Reply:
liveOrClassic has been updated and is therefore now resolved